packages/react/src/components/organisms/FooterSlim/index.js

```diff
@@ -16,60 +16,77 @@ import IconPhone from 'MayflowerReactBase/Icon/IconPhone';
 // eslint-disable-next-line import/no-unresolved
 import IconLaptop from 'MayflowerReactBase/Icon/IconLaptop';
 
-const FooterSlim = (props) => (
-  <footer className="ma__footer-slim" id="footer">
-    <div className="ma__footer-slim__container ma__container">
-      {props.siteLogo}
-      <div className="ma__footer-slim__container__inner">
-        <div className="ma__footer-slim__info">
-          <div className="ma__footer-slim__title">{props.title }</div>
-          <p>{props.description }</p>
-          <p className="ma__footer-slim__copyright">
-            &copy;
-            {' '}
-            {moment().year()}
-            {' '}
-            Commonwealth of Massachusetts
-          </p>
+const FooterSlim = ({
+  title, description, siteLogo, stackedLogo = false, links = null, contact = null
+}) => {
+  const today = new Date();
+  const year = today.getFullYear();
+
+  const logoWrapperClasses = classNames({
+    'ma__footer-slim__container__logos': true,
+    'ma__footer-slim__container__logos--stacked': stackedLogo
+  });
+
+  const innerWrapperClasses = classNames({
+    'ma__footer-slim__container__inner': true,
+    'ma__footer-slim__container__inner--stacked': stackedLogo
+  });
+
+  return(
+    <footer className="ma__footer-slim" id="footer">
+      <div className="ma__footer-slim__container ma__container">
+        <div className={logoWrapperClasses}>
+          {siteLogo}
```

Comment:
I think it would be better to make siteLogo a component instead of just a variable. That way if someone wants to use a component they made, they can do so.
```suggestion
          <SiteLogo />
```